frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -513,7 +523,7 @@ const ManagePolicyPage = ({
 
       // update changed policies calendar events enabled
       const changedPolicies = formData.policies.filter((formPolicy) => {
-        const prevPolicyState = teamPolicies?.find(
+        const prevPolicyState = policiesAvailableToAutomate?.find(
```

Comment:
I think `policiesAvailableToAutomate` will never be undefined now. If so, it would be safe to remove the `?` optional chaining.